kalite/distributed/static/js/distributed/video/views.js

```diff
@@ -68,12 +78,38 @@ var VideoPlayerView = ContentBaseView.extend({
                 "playbackRates": [0.5, 1, 1.25, 1.5, 2],
                 "html5": {
                     nativeTextTracks: false
+                },
+                "techOrder": this.flash_only ? ["flash"] : ["html5", "flash"],
+                flash: {
+                    swf: window.sessionModel.get("STATIC_URL") + "js/distributed/video/video-js.swf"
                 }
             };
             if( this.data_model.get("content_urls").thumbnail ) {
                 video_player_options['poster'] = this.data_model.get("content_urls").thumbnail;
             }
             this.player = window.player = _V_(player_id, video_player_options);
+            this.player.ready(function() {
```

Comment:
May as well delete this codeblock if unused. Without the callback inside, it is pointless.